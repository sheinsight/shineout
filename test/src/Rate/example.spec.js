import exampleTest from '../../example'

describe('Rate[snapshot]', () => {
  exampleTest('Rate')
})
