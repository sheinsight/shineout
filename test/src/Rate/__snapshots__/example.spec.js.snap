// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: Rate[site/pages/components/Rate/example-01-base.tsx] 1`] = `
<div>
  <div>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
  </div>
  <div>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span />
  </div>
</div>
`;

exports[`Snapshot test: Rate[site/pages/components/Rate/example-01-half.tsx] 1`] = `
<div>
  <div>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
  </div>
  <div>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span />
  </div>
</div>
`;

exports[`Snapshot test: Rate[site/pages/components/Rate/example-02-color.tsx] 1`] = `
<div>
  <div>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
  </div>
  <div>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span />
  </div>
</div>
`;

exports[`Snapshot test: Rate[site/pages/components/Rate/example-04-max.tsx] 1`] = `
<div>
  <div>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
  </div>
  <div>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span />
  </div>
</div>
`;

exports[`Snapshot test: Rate[site/pages/components/Rate/example-05-size.tsx] 1`] = `
<div>
  <div>
    <div>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
    </div>
    <div>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span />
    </div>
  </div>
  <br />
  <div>
    <div>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
    </div>
    <div>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span />
    </div>
  </div>
  <br />
  <div>
    <div>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
      <span>
        <i />
      </span>
    </div>
    <div>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span>
        <div />
        <span>
           
        </span>
      </span>
      <span />
    </div>
  </div>
</div>
`;

exports[`Snapshot test: Rate[site/pages/components/Rate/example-06-text.tsx] 1`] = `
<div>
  <div>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
  </div>
  <div>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      very good
    </span>
  </div>
</div>
`;

exports[`Snapshot test: Rate[site/pages/components/Rate/example-07-disabled.tsx] 1`] = `
<div>
  <div>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
  </div>
  <div>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <span>
        <i />
      </span>
    </span>
    <span />
    <span />
  </div>
</div>
`;

exports[`Snapshot test: Rate[site/pages/components/Rate/example-08-face.tsx] 1`] = `
<div>
  <div>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
  </div>
  <div>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <i />
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span />
  </div>
</div>
`;

exports[`Snapshot test: Rate[site/pages/components/Rate/example-09-array.tsx] 1`] = `
<div>
  <div>
    <span>
      A
    </span>
    <span>
      B
    </span>
    <span>
      C
    </span>
    <span>
      D
    </span>
    <span>
      E
    </span>
  </div>
  <div>
    <span>
      <div />
      <span>
        A
      </span>
    </span>
    <span>
      <div />
      <span>
        B
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span />
  </div>
</div>
`;

exports[`Snapshot test: Rate[site/pages/components/Rate/example-10-clearable.tsx] 1`] = `
<div>
  <div>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
    <span>
      <i />
    </span>
  </div>
  <div>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span>
      <div />
      <span>
         
      </span>
    </span>
    <span />
  </div>
</div>
`;
